src/components/editor.js

```diff
@@ -202,8 +202,6 @@ class Editor extends React.Component {
   onChange = (state) => {
     if (state == this.state.state) return
 
-    state = this.onBeforeChange(state)
```

Comment:
Yes we can probably remove the `onBeforeChange` since we are no longer using it (the schema is doing the job).

If we remove it, we have to move the normalization with the plugins schemas to the react component itself.